lib/src/main/java/com/cloudbees/groovy/cps/Continuable.java

```diff
@@ -185,26 +166,6 @@ public Outcome call() {
         });
     }
 
-    /**
-     * Sets a super-interrupt.
-     *
-     * <p>
-     * A super interrupt works like {@link Thread#interrupt()} that throws
-     * {@link InterruptedException}. It lets other threads interrupt the execution
-     * of {@link Continuable} by making it throw an exception.
```

Comment:
Huh. I guess this was not in fact used to implement the **Pause** function in Pipeline?

Reply:
Yeah, this was a bit surprising to me. I think that the safepoint mechanism is currently all that is being used. We could keep this around it case it could be used from `CpsThread` to optimize the time between initiating a pause and the program actually pausing, but we could always bring it back if there is some reason to work on that in the future.

Reply:
YAGNI